validators/terravaloper1epeqf5t6kj7wdr3zwqu38z4wayay3e6vp9s5m2/profile.json

```diff
@@ -0,0 +1,7 @@
+{
+  "contact": {
+    "email": "support@moon-runners.net",
+    "telegram": "@mrunners_val"
```

Comment:
Add a comma after this line to maintain JSON format.  After that, you'll be good to go!